bmt/toolkit.py

```diff
@@ -724,6 +724,52 @@ def is_predicate(self, name: str) -> bool:
         """
         return 'related to' in self.get_ancestors(name)
 
+    @lru_cache(CACHE_SIZE)
+    def is_mixin(self, name: str) -> bool:
+        """
+        Determines whether the given name is the name of a mixin
+        in the Biolink Model. An element is a mixin if one of its properties is "is_mixin:true"
+
+        Parameters
+        ----------
+        name: str
+            The name or alias of an element in the Biolink Model
+
+        Returns
+        -------
+        bool
+            That the named element is a valid mixin in Biolink Model
+        """
+        is_mixin = "false"
+        element = self.get_element(name)
+        if element:
+            is_mixin = element.mixin if isinstance(element, Definition) else None
+        return is_mixin
+
+    @lru_cache(CACHE_SIZE)
+    def has_inverse(self, name: str) -> bool:
+        """
+        Determines whether the given name is a predicate and if that predicate has an inverse defined
+        in the Biolink Model. An element is a predicate if it descends from
+        `related to`
+
+        Parameters
+        ----------
+        name: str
+            The name or alias of an element in the Biolink Model
+
+        Returns
+        -------
+        bool
+            That the named element is a valid mixin in Biolink Model
+        """
+        has_inverse = False
+        element = self.get_element(name)
+        inverse_name = element.inverse if isinstance(element, SlotDefinition) else None
+        if inverse_name is not None:
+            has_inverse = True
+        return bool(has_inverse)
```

Comment:
Again, this is not necessary since the actual value in `element.inverse` is already a boolean

Reply:
done!